charts/k8ez/values.yaml

```diff
@@ -74,6 +85,24 @@ configData: {}
 # Set secret data key-value pairs that will be loaded into env variables
 secretData: {}
 
+# List of files to load into configmaps.  Supports glob patterns, all files that match will be put into the configmap
+configFiles: []
+  # - fileGlob: files/config.toml
+  #   configMapName: config.toml
+
+# List of extra annotations to be injected into pod templates
+extraAnnotations: {}
```

Comment:
Should the extra-annotations be nested under cronjobs, statefulsets, jobs, etc?

Reply:
i could go either way on it. I suppose ideally you would have a global set of them in the root of the values, then override them with resource specific annotations if you want.  In the use case I have been working with we have been using it for the vault agent, and all those resources need to get the same secrets so the global way was nice.  Not sure if we want to implement further right now or not

Reply:
We can circle back to it later.